### ISSUE TITLE
Many volumes or items attached to a work takes a long time to display

### ISSUE THREAD

**Issue by pollecuttn** [wellcomecollection/wellcomecollection.org#8539]
**What**

https://wellcomecollection.org/works/h4t88h83

Under 'Where to find it' click the link '+235 more'. It takes at least 15 seconds to display the large number of volumes and makes it appear that the link is broken or doesn't do anything.

(Reported by email by Ross 26/9/22)

**Where**

https://wellcomecollection.org/works/h4t88h83
(tested on Chrome on Windows)

**TODO**
- [ ] find out how much a problem this is (above what number of attached items do we start to see a performance issue? How many works do we have like that?)
- [ ] investigate if we can improve the display and/or performance for works that have many items attached

**Comment by alexwlchan** [comment]
I've fixed this for lists of items that can't be requested online, but I suspect the issue will persist for works with lots of items that can be requested online.

**Comment by alexwlchan** [comment]
Yup, still an issue for works with lots of online items, see https://wellcomecollection.org/works/bmc9zstd

**Comment by alexwlchan** [comment]
It does eventually load and it only affects a few items, and I'm not sure I can do much about this without a substantial refactor. One to revisit on the works page redesign, which might affect how we render items in a bigger way.

**Comment by pollecuttn** [comment]
Closing as no longer a problem
